src/com/owncloud/android/ui/TextDrawable.java

```diff
@@ -1,52 +1,126 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Andy Scherzinger
+ * @author Tobias Kaminsiky
+ * Copyright (C) 2016 ownCloud Inc.
+ * <p/>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ * <p/>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p/>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 package com.owncloud.android.ui;
 
 import android.graphics.Canvas;
 import android.graphics.Color;
 import android.graphics.ColorFilter;
 import android.graphics.Paint;
 import android.graphics.PixelFormat;
-import android.graphics.Rect;
 import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+
+import com.owncloud.android.utils.BitmapUtils;
+
+import java.io.UnsupportedEncodingException;
+import java.security.NoSuchAlgorithmException;
 
 /**
- * Created by tobi on 24.05.15.
+ * A Drawable object that draws text (1 character) on top of a circular/filled background.
  */
 public class TextDrawable extends Drawable {
```

Comment:
That's better, but makes me wonder "why should I give a full name to print a single character" ? :D 

Maybe something like ```PrettyTextDrawable``` or ```PrettyRoundDrawable```, or like that?